utils.php

```diff
@@ -149,3 +150,21 @@ function normalize_term($term)
     }
     return ($year - 1) . "9";
 }
+
+/**
+ * Function checks if user_id and utorid match. If not, returns 401 error
+ *
+ * @throws 401 Unauthorized Access
+ */
+function verify_user_id($parameters)
+{
+    if (isset($parameters['user_id']) && $parameters['user_id'] != null) {
```

Comment:
Don't do this check. They shouldn't be executing anything if their `user_id` is null.

Reply:
I just want to double check. When making the request to get user id from shibboleth, the user is not yet passed in. So I can exclude the check when user is null but still include the isset check?

Reply:
You don't need any of the isset tests.